.gitattributes

```diff
@@ -8,3 +8,4 @@ ffi/diplomat/cpp/include/** linguist-generated=true
 ffi/diplomat/cpp/docs/** linguist-generated=true
 ffi/diplomat/js/include//** linguist-generated=true
 ffi/diplomat/js/docs/** linguist-generated=true
+*.rs.data linguist-language=Rust
```

Comment:
FWIW, this doesn't actually help from the POV of rust-anayzer. I spoke with the rust-analyzer folks and they basically said that they have to parse every file in the crate either way, but they're patching their parser to not crash here https://github.com/rust-lang/rust-analyzer/issues/13788

Reply:
It doesn't stop r-a reading them. But they aren't proper Rust files, so I gave them a special suffix.

Reply:
I've long thought that they should have a different suffix from the rest of the Rust files in the repository. `.rs.data` seems ok; I was originally thinking something with `.inc` since that's a common extension for include-only files.

Reply:
Yeah that's totally fine, I just wanted to note that this avenue isn't a useful one for fixing rust-analyzer.